COLLABORATOR_GUIDE.md

```diff
@@ -432,28 +476,33 @@ The TSC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
-* Please never use GitHub's green ["Merge Pull Request"](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-on-github) button.
-  * If you do, please force-push removing the merge.
-  * Reasons for not using the web interface button:
-    * The merge method will add an unnecessary merge commit.
-    * The squash & merge method has been known to add metadata to the
-    commit title (the PR #).
-    * If more than one author has contributed to the PR, keep the most recent
-      author when squashing.
-
-Review the commit message to ensure that it adheres to the guidelines outlined
-in the [contributing](./doc/guides/contributing/pull-requests.md#commit-message-guidelines) guide.
-
-Add all necessary [metadata](#metadata) to commit messages before landing.
-
-See the commit log for examples such as
-[this one](https://github.com/nodejs/node/commit/b636ba8186) if unsure
-exactly how to format your commit messages.
+1. Never use GitHub's green ["Merge Pull Request"][] button. Reasons for not
+   using the web interface button:
+   * The merge method will add an unnecessary merge commit.
+   * The squash & merge method has been known to add metadata to the commit
+     title (the PR #).
+   * If more than one author has contributed to the PR, keep the most recent
```

Comment:
Is this point a bit confusing? If this is an advice, should we get it out of "Reasons for not using..." part? If this is a green button artifact, should the "keep" become "keeps" and should the culprit method be mentioned?

Reply:
In fact, I'd strictly be in favor of keeping code by authors who are _not_ core collaborators in PRs even if they are not the latest. I think that if there are multiple contributors we should keep commits from all of them (perhaps with the same commit message)

Reply:
It requires judgment, particularly around how significant the contribution is. If someone contributed something significant, squashing commits in a way that removes them as an author should certainly be avoided. That can look like (or can actually be) stealing credit for someone else's work. On the other hand, we do have to make sure that all tests pass with each commit, and sometimes squashing is the most reasonable way to achieve that.

Reply:
I see the point that this is confusing but I would really like to keep this out of the scope of this PR. I did not change this part, so I guess that is fine?

Reply:
Oh, yeas, hadn't noticed that this content has not changed and this is just a formatting/white-space change to this part. Yeah, the comments should be addressed in another PR IMO.